libcxx/docs/Hardening.rst

```diff
@@ -72,6 +74,10 @@ to control the level by passing **one** of the following options to the compiler
    pre-built components. Most libc++ code is header-based, so a user-provided
    value for ``_LIBCPP_HARDENING_MODE`` will be mostly respected.
 
+In some cases, users might want to override the default assertion semantic.
```

Comment:
```suggestion
In some cases, users might want to override the assertion semantic used by the library.
```

IMO the word `default` is confusing here -- the default semantic is something that the vendors would set, not the users.